### DISCUSSION TITLE
How to run vscode debugger on a Tsed project?

### DISCUSSION THREAD

**Discussion by LefanTan** [TypedProject/ts-express-decorators#2196] [Q&A] [answered]
Hey everyone, this framework has been amazing thus far, but I can't seem to find any tutorials/documentation on debugging a Tsed application. Does any know how to start?

**Comment by Romakita** [comment, marked as answer]
Hello @LefanTan 
Debug a tsed app is the same as debugging node.js application. You can find many tutorials on google for that and depending on your IDE.
